libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/SqlParser.java

```diff
@@ -27,6 +27,7 @@
 import com.oceanbase.tools.dbbrowser.parser.result.ParseSqlResult;
 import com.oceanbase.tools.sqlparser.obmysql.OBLexer;
 import com.oceanbase.tools.sqlparser.obmysql.OBParser;
+import com.oceanbase.tools.sqlparser.util.TimeoutTokenStream;
```

Comment:
same as above

Reply:
TimeoutTokenStream is defined in db-browser originally and I move it to the ob-sql-parser, to that the import is changed~